src/controllers/commands/CommandController.cpp

```diff
@@ -2936,6 +2937,130 @@ void CommandController::initialize(Settings &, Paths &paths)
+            }
+            break;
+
+            case Error::UserMissingScope: {
+                // TODO(pajlada): Phrase MISSING_REQUIRED_SCOPE
+                errorMessage += "Missing required scope. "
+                                "Re-login with your "
+                                "account and try again.";
+            }
+            break;
+
+            case Error::UserNotAuthorized: {
+                // TODO(pajlada): Phrase MISSING_PERMISSION
+                errorMessage += "You don't have permission to "
+                                "perform that action.";
+            }
+            break;
+
+            case Error::Unknown: {
+                errorMessage += "An unknown error has occurred.";
+            }
+            break;
+        }
+        return errorMessage;
+    };
+
+    this->registerCommand(
+        "/vips",
+        [formatVIPListError](const QStringList &words,
+                             auto channel) -> QString {
+            switch (getSettings()->helixTimegateVIPs.getValue())
+            {
+                case HelixTimegateOverride::Timegate: {
+                    if (areIRCCommandsStillAvailable())
+                    {
+                        return useIRCCommand(words);
+                    }
+
+                    // fall through to Helix logic
+                }
+                break;
+
+                case HelixTimegateOverride::AlwaysUseIRC: {
+                    return useIRCCommand(words);
+                }
+                break;
+
+                case HelixTimegateOverride::AlwaysUseHelix: {
+                    // do nothing and fall through to Helix logic
+                }
+                break;
+            }
+
+            auto *twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+            if (twitchChannel == nullptr)
+            {
+                channel->addMessage(makeSystemMessage(
+                    "The /vips command only works in Twitch channels"));
+                return "";
+            }
+
+            auto currentUser = getApp()->accounts->twitch.getCurrent();
+            if (currentUser->isAnon() ||
+                currentUser->getUserId() != twitchChannel->roomId())
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Due to Twitch restrictions, this command can only be used "
+                    "by "
+                    "the broadcaster. To see the list of VIPs you must use the "
+                    "Twitch website."));
+                return "";
+            }
+
+            getHelix()->getChannelVIPs(
+                twitchChannel->roomId(),
+                [channel, twitchChannel](const std::vector<HelixVip> &vipList) {
+                    if (vipList.empty())
+                    {
+                        makeSystemMessage(
+                            "This channel does not have any VIPs.");
+                        return;
+                    }
```

Comment:
this doesn't work
![image](https://user-images.githubusercontent.com/41973452/194740141-568819ab-497a-4e1d-be21-00347803492f.png)

![image](https://user-images.githubusercontent.com/41973452/194739966-bc04e117-1548-4c14-9f1e-8c01595b2110.png)

Reply:
Fixed